waku/v2/utils/time.nim

```diff
@@ -0,0 +1,24 @@
+## Contains types and utilities for timestamps.
+
+{.push raises: [Defect].}
+
+import sqlite3_abi
+
+type Timestamp* = int64 
+
+const TIMESTAMP_TABLE_TYPE* = "INTEGER"
+
+proc getNanosecondTime*[T](timeInSeconds: T): Timestamp = 
+  var ns = Timestamp(timeInSeconds*100000000)
+  return ns
+
+proc getMicrosecondTime*[T](timeInSeconds: T): Timestamp = 
+  var us = Timestamp(timeInSeconds*1000000)
+  return us
+
+proc getMillisecondTime*[T](timeInSeconds: T): Timestamp = 
+  var ms = Timestamp(timeInSeconds*1000)
+  return ms
+
+proc column_timestamp*(a1: ptr sqlite3_stmt, iCol: cint): int64 =
```

Comment:
Thanks for defining this proc, though I think it is also ok to use the `sqlite3_column_int64` directly in the code. I'd like to hear @jm-clius opinion as well

Reply:
[Here](https://github.com/status-im/nim-waku/pull/842#discussion_r803178993) a related comment to this.

Reply:
Fair enough, I cannot see any issue with having a separate proc `column_timestamp`